README.md

```diff
@@ -24,8 +26,8 @@ function add(x, y) {
   return x + y
 }
 ```
+ES5
 
```

Comment:
You didn't answer the other 2 questions.

This is anonymous
and you call it as `add(3,4);`